components/swatchgroup/stories/template.js

```diff
@@ -33,7 +38,48 @@ export const Template = ({
 		${items.map((swatch) => Swatch({
 			size,
 			rounding,
+			isDisabled: isDisabled,
+			isSelected: isSelected,
+			borderStyle,
```

Comment:
Is there a way to pass this borderStyle to just the Swatches that _also_ have their borderStyle defined? Toggling borderStyle, disabled, selected, all affect every swatch in a swatchgroup currently. That's not how I would imagine it works in practice, but maybe it's not a big deal that we can do that in storybook. Thoughts?

I was thinking something like: `borderStyle: swatch.borderStyle === borderStyle ? borderStyle : undefined` but I couldn't quite get that to work (I didn't spend much time on that solution either).

Reply:
I think this is passing down the border style, if you set it in the args in swatchgroup.stories.js, for instance:

```
		items: [
			{swatchColor: "rgb(184, 109, 70)", borderStyle: "noBorder"},
			{swatchColor: "rgb(240, 56, 35)",},
			{swatchColor: "rgb(212, 91, 0)",},
			{swatchColor: "rgb(175, 116, 0)",},
			{swatchColor: "rgb(154, 123, 77)",},
			{swatchColor: "rgb(114, 137, 0)"},
			{swatchColor: "rgb(72, 144, 20)"},
			{swatchColor: "rgb(7, 147, 85)"},
			{swatchColor: "rgb(9, 144, 120)"},
			{swatchColor: "rgb(10, 141, 153)"},
			{swatchColor: "rgb(18, 134, 205)"},
			{swatchColor: "rgb(75, 117, 255)"},
			{swatchColor: "rgb(122, 106, 253)"},
			{swatchColor: "rgb(166, 92, 231)"},
			{swatchColor: "rgb(200, 68, 220)"},
			{swatchColor: "rgb(228, 52, 163)"},
			{swatchColor: "rgb(240, 45, 110)", borderStyle: "noBorder"},
		],
```

Those two noBorder items will show without a border. Were you looking for something different from that? (Also, not sure if this is something you're still looking to support?)

Reply:
Now that we've changed up the light border stuff, I don't think there's a need for this. I was trying to figure out how to do the individual low-contrast swatches, to prevent the incorrect borders if the controls are used but, I don't think it's necessary after our talk anymore.